### PR TITLE
Feat/connect explorer theme fork buildless

### PR DESCRIPTION
**Pull request by jvaclavik**
## Related Issue

Resolve

### CODE REVIEW

**Comment by martykan** [comment]
@SocketSecurity ignore-all

File: packages/connect-explorer-nextra/src/pages/index.mdx
Comment (martykan) [inline packages/connect-explorer-nextra/src/pages/index.mdx:1]: This is probably unintentional right?

File: packages/connect-explorer-theme/LICENSE
Comment (martykan) [inline packages/connect-explorer-theme/LICENSE:1]: Could there be anything from the legal side? Seems to be the first case where we have licensed code in the monorepo

File: packages/utxo-lib/tsconfig.json [hunk 1]
```diff
@@ -1,8 +1,6 @@
 {
     "extends": "../../tsconfig.base.json",
-    "compilerOptions": {
-        "outDir": "./libDev"
-    },
+    "compilerOptions": { "outDir": "./libDev" },
```
Comment (martykan) [inline packages/utxo-lib/tsconfig.json]: Probably unintentional right? This should be reverted

**Review by martykan — CHANGES_REQUESTED** [review]

File: packages/connect-explorer-theme/LICENSE
Comment (jvaclavik) [reply packages/connect-explorer-theme/LICENSE:1]: I discussed it with @MiroslavProchazka  and it should be OK. We have info about the autor and licence in package.json, so it should be correct.

File: packages/utxo-lib/tsconfig.json (hunk 1 above)
Comment (jvaclavik) [reply packages/utxo-lib/tsconfig.json]: Done

File: packages/connect-explorer-nextra/src/pages/index.mdx
Comment (jvaclavik) [reply packages/connect-explorer-nextra/src/pages/index.mdx:1]: Can't see any change in index.mdx, so I guess it's fixed 🤷‍♂️

**Review by martykan — APPROVED** [review]
LGTM just needs autosquash

**Pull request merged**